src/Microsoft.WSMan.Management/resources/WsManResources.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2008 Microsoft Corporation
+# Copyright (c) Microsoft Corporation
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."